composer.json

```diff
@@ -94,7 +94,8 @@
         "facebook/facebook-instant-articles-sdk-php": "v1.5.*",
         "takeit/amp-html-bundle": "^0.2.0@dev",
         "asm89/twig-cache-extension": "^1.3",
-        "doctrine/doctrine-migrations-bundle": "^1.0"
+        "doctrine/doctrine-migrations-bundle": "^1.0",
+        "layershifter/tld-extract": "^1.2"
```

Comment:
add this also to composer.json in  multitenancy component or is this lib auctually needed needed for this simple use case? you can easily extract domain using own php implementation using explode